spec/requests/errors_controller_spec.rb

```diff
@@ -0,0 +1,96 @@
+RSpec.describe ErrorsController, type: :request do
+  include ApplicationHelper
+
+  subject(:rendered) { make_request && response }
+
+  let(:json_response) { JSON.parse(rendered.body) }
```

Comment:
I always think it would be nice to move this type of method in a global helper but then depends what we are passing into the body part

Reply:
Can pass body here i guess as use it below